pom.xml

```diff
@@ -108,12 +108,38 @@ SAX2 and Stax2 APIs
             <scope>provided</scope>
        -->
             <optional>true</optional>
+            <exclusions>
```

Comment:
Maybe add a comment to indicate why these are needed?

Reply:
I've added some comments.
To summarize:

1. Don't include transient `isorelax:isorelax` dependency that lacks source code on [Maven Central](https://repo1.maven.org/maven2/isorelax/isorelax/20030108/), since this implies questionable provenance.
2. Don't include transient `xml-apis:xml-apis` dependency that is repackaging XML APIs (SAX, DOM, etc.) that are built into the JDK itself.
3. Don't include transient `xerces:xercesImpl` dependency since all modern JDKs already include it.
4. Manually specify `com.sun.xml.bind.jaxb:isorelax` since that includes source code on [Maven Central](https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb/isorelax/20090621/), and matches the version that was shaded in previous Woodstox releases.

Excluding the transient `xml-apis:xml-apis` & `xerces:xercesImpl` dependencies can also help ensure that unit tests don't end up falsely relying on features/quirks of these jar's instead of relying on the versions that are included in the base JDK.

Reply:
Given that there are sort of 2 separate things -- exclusion of XercesImpl, JDK APIs -- and then bigger part of iso-relax, would it make sense to separate these into different PRs?

To me former sounds safer in general so could go in first, and perhaps released as 6.6.2; and second bigger part separately, ad 6.7.0.
(and both merge up for upcoming 7.0.0).

Reply:
Ahhh. Ok, yes, there's not that much point since these we knew unwanted dependencies.
Not something that was already leaked like I assumed for some reason.

So let's go with just this one.